### DISCUSSION TITLE
403 Forbidden

### DISCUSSION THREAD

**Discussion by YasMax91** [Laravel-Backpack/community-forum#1064] [Q&A (Help)] [answered]
If you follow the link https://some-site.com/admin, it doesn't redirect to https://some-site.com/admin/dashboard, but gives error 403. The same situation occurs when clicking the "Cancel" button in the profile.
I am using Laravel 11. In Laravel 10 this did not happen. Perhaps I need additional configuration of nginx

**Comment by jcastroa87** [comment]
Hello @YasMax91 

Can you check config in base.php

```
'route_prefix' => 'admin',
'setup_auth_routes' => true,
'setup_dashboard_routes' => true,
'setup_my_account_routes' => true,
```

Did you try this in a fresh install? Because i try in my side and is working normal.

Please share the version too.

```
php artisan backpack:version
```

Cheers.

**Reply by YasMax91** [reply, marked as answer]
All the configs are correct - just as you specified. But I just found the problem and fixed it. Unfortunately I didn't have the line "error_page 403 /index.php" written on the server either. Because of this I was thrown to 403 nginx, not 403 page of the project.
Thank you for your help.

But I want to clarify on another issue if it is possible?
What is the correct flow for installing backpack on production environment?

**Reply by jcastroa87** [reply]
Hello, to deploy Backpack in production, make sure yo do this:

1. Set your env params.
```
APP_ENV=production
APP_DEBUG=false
APP_URL=https://yourdomain.com
```
2. Run install command:
```
composer install --optimize-autoloader --no-dev
```
3. Set cache
```
php artisan optimize:clear
php artisan basset:clear
php artisan basset:cache
php artisan optimize
```

Cheers.
